modules/javafx.graphics/src/main/java/com/sun/glass/ui/win/WinTextRangeProvider.java

```diff
@@ -37,7 +37,7 @@
  * This class is the Java peer for GlassTextRangeProvider.
  * GlassTextRangeProvider implements ITextRangeProvider.
  */
-class WinTextRangeProvider {
+public final class WinTextRangeProvider {
```

Comment:
Making both this class and the newly added method public just for the purpose of a unit test is not a pattern we want to use. Instead, leave them as default (package) scope and create a public `WinTextRangeProvideShim` class underneath the shims.